person.rb

```diff
@@ -10,6 +10,11 @@ def initialize(age, name = 'Unknown', parent_permission: true)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    @rentals << rental
   end
```

Comment:
- [ ] Adding rentals should be done by using the Renal class and the person itself should be sent as a parameter to create the new rental achieving the 'has-many' association. Let's fix that

Reply:
Thank you, I did that